phantomgames/Home/CustomCells/CustomCollectionViewCell.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  CustomCollectionViewCell.swift
+//  phantomgames
+//
+//  Created by Aphiwe Shozi on 2024/05/13.
+//
+
+import UIKit
+
+class CustomCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak private var apiImage: UIImageView!
+    
```

Comment:
no marks and empty space between outlets? why??

Reply:
my apologies I misunderstood. Thought marks were for organising code for readability didn't know it applied to such short code. Will apply the changes